src/region/RegionAnnotations.scss

```diff
@@ -6,7 +6,6 @@
     left: 0;
     width: 100%;
     height: 100%;
-    filter: drop-shadow(0 2px 3px rgba(0, 0, 0, .25));
```

Comment:
Removal of this line resolves the performance issues we have when drawing in Firefox.